robottelo/cli/import_.py

```diff
@@ -302,10 +307,10 @@ def config_file_with_tr_data(cls, options=None):
         transition_data = []
         if result.return_code == 0:
             transition_data = [
-                cls.read_transition_csv(ssh.command(command).stdout[:-1])
-                for command
+                cls.read_transition_csv(ssh.command(cmd).stdout[:-1])
+                for cmd
                 in (
-                    u'ls -v ${HOME}/.transition_data/products*'
+                    u'ls -v ${HOME}/.transition_data/products*',
                     u'ls -v ${HOME}/.transition_data/puppet_repositories*'
```

Comment:
Nitpick: Add a trailing comma here too. It'll make future diffs cleaner and make it easier to re-order lines.